{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -19,6 +19,19 @@ repos:
     -   id: pretty-format-json
         args: [--autofix, --no-sort-keys]
     -   id: trailing-whitespace
+-   repo: https://github.com/asottile/blacken-docs
+    rev: v0.4.0
```

Comment:
Should be 0.5.0, no? That would go with blac 19.3b0 as well, it seems. https://github.com/asottile/blacken-docs/commits/master

Reply:
Sorry, probably yes. Will change it. In the README it still says v0.4.0. They advise to fix the black version because black is apperently a "moving target" right now.